infrastructure/kubernetes-demo/README.md

```diff
@@ -10,3 +10,5 @@ Deploy a Hello World Node app to an AWS EKS cluster using Fargate compute.
 3. Follow the instructions from AWS to [set up the AWS Load Balancer Controller](https://docs.aws.amazon.com/eks/latest/userguide/aws-load-balancer-controller.html)
 4. Create a Kubernetes secret for the database endpoint and port from step 1 by running `kubectl create secret generic hello-world-db --from-literal=endpoint='ENDPOINT' --from-literal=port='PORT' -n hello-world`.
 5. Deploy the demo app by running `kubectl apply -f hello-world.yaml`
+
+**NOTE:** The cluster is still in use to run ESS (see [the ESS folder](../ess/)) but the demo app is no longer deployed to save on AWS costs.
```

Comment:
Minor note for the commit mesage rather than the code!

you say;
```
`kubectl delete -f hell-node.yaml` before deleting the manifest file.
```

guessing that was:
```
`kubectl delete -f hello-node.yaml` before deleting the manifest file.
```

Total nitpick, non blocking!